x/sysproxy/doc.go

```diff
@@ -0,0 +1,39 @@
+/*
+Package sysproxy provides a simple interface to set/unset system-wide proxy settings.
+
+# Platform Support
+Currently this package supports desktop platforms only. The following platforms are supported:
```

Comment:
Add an empty line which is required by Godoc: https://tip.golang.org/doc/comment#headings

```suggestion

Currently this package supports desktop platforms only. The following platforms are supported:
```